docs/building-unified.md

```diff
@@ -0,0 +1,70 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## Building Accessibility Insights for Android (Unified)
+
+This document describes how to build and test the Accessibility Insights for Android (the Electron app). Much of the information/code is shared between it and Accessibility Insights for Web (the browser extension); see [Building Accessibility Insights for Web](./building-web.md) for the shared and web-specific guidance.
```

Comment:
Small suggestion to switch to either of:
- "and test ~the~ Accessibility Insights for Android (the Electron app)" 
- "and test the Accessibility Insights for Android **product** (the Electron app)".

This comment also applies to building-web